### PR TITLE
Flutter & Friends 2024

### PR DESCRIPTION
**Pull request by ueman**
Exact dates are still TBA

### CODE REVIEW

File: _conferences/flutter-n-friends-2024.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+location: Stockholm, Sweden
+online: false
+
+date_start: August/September
+date_end:   August/September
+
+cfp:
+  start: TBA
+  end:   TBA
+  site: TBA
```
Comment (marandaneto) [inline _conferences/flutter-n-friends-2024.md]: Maybe this project should support an "estimation date" tag 😅 at least it gets listed.

**Comment by MrAlek** [comment]
I'm happy you were excited enough already in November to assume we would do another run 😃

Dates will be Sep 1-3 🎉 Website updates in a few days 🙌

File: _conferences/flutter-n-friends-2024.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+website: https://www.flutterfriends.dev/
+location: Stockholm, Sweden
+online: false
+
+date_start: August/September
+date_end:   August/September
+
+cfp:
+  start: TBA
+  end:   TBA
```
Comment (ueman) [inline _conferences/flutter-n-friends-2024.md]:
```suggestion
  start: 2024-09-01
  end:   2024-09-03
```

File: _conferences/flutter-n-friends-2024.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+name: "Flutter & Friends"
+website: https://www.flutterfriends.dev/
+location: Stockholm, Sweden
+online: false
+
+date_start: August/September
+date_end:   August/September
```
Comment (marandaneto) [inline _conferences/flutter-n-friends-2024.md]:
```suggestion
date_start: 2024-09-01
date_end:   2024-09-03
```

File: _conferences/flutter-n-friends-2024.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+location: Stockholm, Sweden
+online: false
+
+date_start: 2024-09-01
+date_end:   2024-09-03
+
+cfp:
+  start: 2024-09-01
+  end:   2024-09-03
+  site: TBA
```
Comment (marandaneto) [inline _conferences/flutter-n-friends-2024.md]:
```suggestion
  site: https://www.flutterfriends.dev/
```

**Comment by marandaneto** [comment]
Since there's a date, we can merge it and fix the cfp metadata once its out.

**Review by marandaneto — APPROVED** [review]

**Pull request merged**